biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -162,7 +169,7 @@ subroutine FatesPlantRespPhotosynthDrive (nsites, sites,bc_in,bc_out,dtime)
     real(r8) :: mm_ko2             ! Michaelis-Menten constant for O2 (Pa)
     real(r8) :: co2_cpoint         ! CO2 compensation point (Pa)
     real(r8) :: btran_eff          ! effective transpiration wetness factor (0 to 1) 
-    real(r8) :: bbb                ! Ball-Berry minimum leaf conductance (umol H2O/m**2/s)
+    real(r8) :: stomatal_intercept_btran   ! minimum leaf conductance (umol H2O/m**2/s)
```

Comment:
@Qianyuxuan is this the new g0*Bsw? which will go into both stomatal models?

Reply:
Yes. "stomatal_intercept_btran" is the water-stressed g0, which will go into both stomatal models.

Reply:
@serbinsh Yes. "stomatal_intercept_btran" is the water-stressed g0, which will go into both stomatal models.